app/controllers/api/v0/tasks_controller.rb

```diff
@@ -3,6 +3,17 @@ module V0
     class TasksController < ApplicationController
       include Api::V0::Mixins::IndexMixin
       include Api::V0::Mixins::ShowMixin
+      include Api::V0::Mixins::UpdateMixin
+
+      private
+
+      def params_for_list
```

Comment:
These methods shouldn't be needed, the generic ones in `ApplicationController` should handle it